strongbox-web-core/src/main/java/org/carlspring/strongbox/converters/users/AccessModelFormToUserAccessModelDtoConverter.java

```diff
@@ -1,74 +1,78 @@
                                                        .orElseGet(
                                                                () ->
                                                                {
-                                                                   UserStorageDto userStorageDto = new UserStorageDto();
+                                                                   StoragePrivilegesDto userStorageDto = new StoragePrivilegesDto();
                                                                    userStorageDto.setStorageId(
                                                                            repositoryAccess.getStorageId());
-                                                                   userAccessModelDto.getStorages().add(userStorageDto);
+                                                                   userAccessModelDto.getStorageAuthorities().add(userStorageDto);
                                                                    return userStorageDto;
                                                                });
 
-            UserRepositoryDto repository = storage.getRepository(repositoryAccess.getRepositoryId())
+            RepositoryPrivilegesDto repository = storage.getRepositoryPrivileges(repositoryAccess.getRepositoryId())
                                                   .orElseGet(
                                                           () ->
                                                           {
-                                                              UserRepositoryDto userRepositoryDto = new UserRepositoryDto();
+                                                              RepositoryPrivilegesDto userRepositoryDto = new RepositoryPrivilegesDto();
                                                               userRepositoryDto.setRepositoryId(
                                                                       repositoryAccess.getRepositoryId());
-                                                              storage.getRepositories().add(userRepositoryDto);
+                                                              storage.getRepositoryPrivileges().add(userRepositoryDto);
                                                               return userRepositoryDto;
                                                           });
 
             if (StringUtils.isBlank(repositoryAccess.getPath()))
             {
-                repository.getRepositoryPrivileges().addAll(pullPrivilegeDtos(repositoryAccess));
+                repository.getRepositoryPrivileges().addAll(pullPrivileges(repositoryAccess));
                 continue;
             }
 
-            UserPathPrivilegesDto pathPrivileges = repository.getPathPrivilege(repositoryAccess.getPath(),
+            PathPrivilegesDto pathPrivileges = repository.getPathPrivilege(repositoryAccess.getPath(),
                                                                                repositoryAccess.isWildcard())
```

Comment:
Cosmetic: Could you please re-indent the method parameters? Thanks! :)

Reply:
fixed